File: Com.Fubei.Api.OpenApi/Biz/MerchantApi/OpenApiAttribute.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Com.Illuminati.Galileo.Attr;
using Com.Illuminati.Galileo.Constants;

namespace Com.Illuminati.Galileo.Biz.MerchantApi
{
    public class MerchantApiAttribute : PostApiAttribute
    {
        public MerchantApiAttribute()
        {
            Path = "/gateway";
            Category = GalileoApiConfig.Category.MerchantApi;
        }
    }
}

File: Com.Fubei.OpenApi.Vb/Biz/MerchantApi/OpenApiAttribute.vb
Imports Com.Illuminati.Galileo
Imports Com.Illuminati.Galileo.Attr

Namespace Biz.V1
    Public Class MerchantApiAttribute 
        Inherits PostApiAttribute

        Public Sub New()
            Path = "/gateway"
            Category = GalileoApiConfig.Category.MerchantApi
        End Sub
    End Class
End Namespace
